src/analysis/retail/deathknight/frost/Guide.tsx

```diff
@@ -45,13 +45,20 @@ export default function Guide({ modules, events, info }: GuideProps<typeof Comba
               wasting any of the resources granted.
               {info.combatant.hasTalent(talents.HORN_OF_WINTER_TALENT) &&
                 modules.hornOfWinter.guideCastBreakdown}
-              {(info.combatant.hasTalent(talents.EMPOWER_RUNE_WEAPON_SHARED_TALENT) ||
-                info.combatant.hasTalent(talents.EMPOWER_RUNE_WEAPON_FROST_TALENT)) &&
+              {info.combatant.hasTalent(talents.EMPOWER_RUNE_WEAPON_TALENT) &&
                 modules.empowerRuneWeapon.guideCastBreakdown}
+              {/*            THIS WAS THROWING AN ERROR BECAUSE EMPOWER RUNE WEAPON TALENT HAS BEEN MODIFIED   
```

Comment:
Remove this comment. You applied the right fix here to use the single talent id.